pages/dkp/kommander/2.1/install/configuration/index.md

```diff
@@ -75,6 +75,10 @@ apps:
 ...
 ```
 
+### Minimal Kommander installation
+
+You can install Kommander with a bare minimum of applications on a small environment with smaller memory, storage, and CPU requirements like a laptop for testing and demo purposes. Refer to the [Install Kommander on a small environment][small_env] documentation.
```

Comment:
As far as I know, we don't have any documented, easy, way to install Konvoy on a 'laptop'.  I guess you can install Kommander on Kind cluster but this requires additional setup like Metallb...

Rather than try to tackle all those issues with this PR, I suggest we eschew the 'like a laptop' wording for now.

```suggestion
You can install Kommander with a bare minimum of applications on a small environment with smaller memory, storage, and CPU requirements for testing and demo purposes. Refer to the [Install Kommander on a small environment][small_env] documentation.
```